doc/api/fs.md

```diff
@@ -1577,13 +1577,13 @@ _Note: [`fs.watch()`][] is more efficient than `fs.watchFile` and
 `fs.unwatchFile`. `fs.watch` should be used instead of `fs.watchFile` and
 `fs.unwatchFile` when possible._
 
-## fs.write(fd, buffer, offset, length[, position], callback)
+## fs.write(fd, buffer[, offset[, length[, position]]], callback)
 <!-- YAML
 added: v0.0.2
 -->
 
 * `fd` {Integer}
-* `buffer` {String | Buffer}
+* `buffer` {Buffer}
```

Comment:
From the doc changes, this appears to be semver major, in that it removes the `fs.write(fd, string, offset, length, position, callback)` as a supported use. Was that usage not implemented, so its aligning docs with actual implementation?